testr/runner.py

```diff
@@ -7,6 +7,24 @@
 import os
 
 
+# Make a class to wrap sys.stdout to handle requests for isatty.  This is needed
+# to get the right output from pytest when run from FOT MATLAB pyexec.
+# Borrowed from https://github.com/pytest-dev/pytest/issues/5462
+class StdOutWrapper:
+
+    def __init__(self, stdout):
+        self._stdout = stdout
+
+    def __getattr__(self, item):
+        if item == "isatty":
```

Comment:
I don't think this is needed. See the [__getattr__ docs](https://docs.python.org/3/reference/datamodel.html#object.__getattr__). 

Just to be sure I did a test and made an object of this class. Then `stdwrapper.isattry()` definitely just goes through the defined `isattry` object method.

Reply:
OK, I removed the if/else in the __getattr__ and retested.